assets/js/blocks/cart-checkout/cart/full-cart/utils.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { decodeEntities } from '@wordpress/html-entities';
+import { PLACEHOLDER_IMG_SRC } from '@woocommerce/block-settings';
+
+/**
+ * Return a currency value as a number for doing calculations.
+ * Note this doesn't convert into dollars, currency values are in minor units (e.g. cents).
+ *
+ * @param {string} currencyValue Currency value string (in minor unit).
+ * @return {number} The currency value as int (in minor unit).
+ */
+export const getPriceNumber = ( currencyValue ) => {
+	return parseInt( currencyValue, 10 );
+};
+
+/**
+ * Returns a formatted element containing variation details.
+ */
+export const ProductVariationDetails = ( { variation } ) => {
+	const variationsText = variation
+		.map( ( v ) => {
+			if ( v.attribute ) {
+				return `${ decodeEntities( v.attribute ) }: ${ decodeEntities(
+					v.value
+				) }`;
+			}
+			// Support for product attributes with no name/key
+			return `${ decodeEntities( v.value ) }`;
+		} )
+		.join( ' / ' );
+
+	return (
+		<div className="wc-block-cart-item__product-attributes">
+			{ variationsText }
+		</div>
+	);
+};
+
+/**
+ * Formats and returns an image element.
+ */
+export const ProductImage = ( { image = {} } ) => {
+	const imageProps = {
+		src: image.src || PLACEHOLDER_IMG_SRC,
+		alt: image.alt || '',
```

Comment:
You could call `decodeEntities` here
```suggestion
		alt: decodeEntities( image.alt ) || '',
```
this way you can remove `alt={ decodeEntities( imageProps.alt ) }` from the next line.